### PR TITLE
feat: add tax details on public event tickets

### LINKED ISSUE #2082: No Tax Information being shown on public tickets
**Describe the bug**

Even after setting up tax information for an event, we can't see it being shown up on the event's ticket page or anywhere later in ticket process.

**To Reproduce**
1. Set taxes in an event
2. Go to event page
3. Scroll down to Tickets

**Expected behavior**

A separate Tax Information must be shown which shows the tax rate and related information

**Screenshots**

taxinfo
ticketwithouttax

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 16.04 LTS
 - Browser [e.g. chrome, safari] Chrome
 
**Additional context**

Can be worked upon by anyone

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #2082 

#### Short description of what this resolves:
- displays tax information on public tickets
- adds taxedAmount property in ticket model for ease of tax calculation
- changed the design of order summery widget

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by uds5501** [comment]
Images after the redesign : 

Public ticket page:
tick1

New Order Page:
Screenshot from 2019-06-04 23-01-36

Ps : I will create a new event in deployment and provide discount codes and event name for testing.

**Comment by uds5501** [comment]
@shreyanshdwivedi @mrsaicharan1 @kushthedude please review.

File: app/models/ticket.js [hunk 1]
```diff
@@ -44,5 +44,13 @@ export default ModelBase.extend({
 
   itemTotal: computed('price', 'quantity', function() {
     return this.price * this.quantity;
+  }),
+
+  taxedAmount: computed('event.tax', function() {
+    let taxType = this.event.get('tax.isTaxIncludedInPrice');
+    if (!taxType) {
+      return (1 + this.event.get('tax.rate')/100)*this.price;
```
Comment (niranjan94) [inline app/models/ticket.js]: space around `*`

File: app/components/public/ticket-list.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ export default Component.extend(FormMixin, {
 
   total: computed('tickets.@each.orderQuantity', 'tickets.@each.discount', function() {
     return sumBy(this.tickets.toArray(),
-      ticket => (ticket.getWithDefault('price', 0) - ticket.getWithDefault('discount', 0)) * ticket.getWithDefault('orderQuantity', 0)
+      ticket => (ticket.getWithDefault('taxedAmount', 0) - ticket.getWithDefault('discount', 0)) * ticket.getWithDefault('orderQuantity', 0)
```
Comment (niranjan94) [inline app/components/public/ticket-list.js]: `ticket.getWithDefault('taxedAmount', 0)` could be `ticket.taxedAmount || 0` 

`ticket.getWithDefault('discount', 0)` could be `ticket. discount || 0` 

`ticket.getWithDefault('orderQuantity', 0)` could be `ticket.orderQuantity || 0`

**Comment by mariobehling** [comment]
Sorry to mention this that way, but this implementation is completely confusing. Please investigate how Eventbrite does this and add the info how Eventbrite does this to the issue itself. Then, please make a new PR with the same implementation. Thank you. Closing this therefore.

**Pull request closed without merging**